test/src/kbasesearchengine/test/main/SearchMethodsTest.java

```diff
@@ -277,5 +294,66 @@ private void searchTypesCheckMatchFilter(
         // if we want to check search time, need to mock a Clock. Don't bother for now.
         // assertThat("incorrect objects", res.getSearchTime(), is(20));
     }
-    
+
+
+
+
+    @Test
+    public void searchObjectResult() throws Exception {
```

Comment:
There's at least 4 different cases that need to be covered in SearchMethods diff - I think this only covers one?

Reply:
there is getObjectsByIds. <= I can add this in the rare case someone desperately wants the guids back highlighted

        //searchIds is a wrapper around queryHits and does not return object data and so will not be highlighted
        //searchTypes returns the number of items per type that. Highlight is not added b/c does not call buildObjects

Reply:
We talked this one out, was talking about this line

`ret.objectHighlight = toBool(pp.getIncludeHighlight()) && !idsOnly;`